### PR TITLE
Ports the smuggler's satchel

### PR DESCRIPTION
**Pull request by JacobLee1**
Used for hiding your dirty stolen goods and/or traitor items.

Has an issue where using the t-ray scanner allows one to access it while it's under the plating; this type of issue is present on /tg/ as well and exists with all underfloor objects. Pipes can be ripped up without taking the plating off, for example.

### CODE REVIEW

File: data/mode.txt [hunk 1]
```diff
@@ -1 +1 @@
-secret
+extended
```
Comment (Glloyd) [inline data/mode.txt]: honk

**Comment by Glloyd** [comment]
Yeah, not much we can do about the T-ray thing. I'm going to go make dinner then take a look at it.

**Comment by Nienhaus** [comment]
Gosh Jacob. Why'd you have to say anything about the t-ray scanner. How can I cheat now!

**Comment by Glloyd** [comment]
:+1:

**Comment by Glloyd** [comment]
That branch name though.

**Pull request merged**